libs/core/src/service/activity_service.rs

```diff
@@ -0,0 +1,71 @@
+use crate::CoreState;
+use crate::LbResult;
+use crate::Requester;
+use db_rs::DbError;
+use lockbook_shared::document_repo::DocActivityMetrics;
+use lockbook_shared::document_repo::DocEvent;
+use lockbook_shared::document_repo::StatisticValueRange;
+use lockbook_shared::document_repo::Stats;
+use uuid::Uuid;
+
+impl<Client: Requester> CoreState<Client> {
+    pub(crate) fn suggested_docs(&mut self) -> LbResult<Vec<Uuid>> {
+        let mut scores = self
```

Comment:
with the current implemtnation, I'm pretty sure `score == vec![]` no matter what the current state of things is. And in some situations you're going to panic. You need some tests in these parts to catch these mistakes.